pkg/sqlqueries/SQL/mto/milestone/get_by_model_plan_id_no_linked_solution_LOADER.sql

```diff
@@ -0,0 +1,40 @@
+WITH QIDS AS (
+    SELECT UNNEST(CAST(:model_plan_ids AS UUID[])) AS model_plan_id
+)
+
+SELECT
+    m.id,
+    m.model_plan_id,
+    m.mto_common_milestone_key,
+    m.mto_category_id,
+    COALESCE(m.name, cm.name) AS name,
+    m.facilitated_by,
+    m.need_by,
+    m.status,
+    m.risk_indicator,
+    m.is_draft,
+    m.created_by,
+    m.created_dts,
+    m.modified_by,
+    m.modified_dts
+FROM mto_milestone AS m
+/* Fallback name from mto_common_milestone if m.name is null */
+LEFT JOIN mto_common_milestone AS cm
+    ON m.mto_common_milestone_key = cm.key
+
+/* LEFT JOIN to see if this milestone is directly linked to any solution */
+LEFT JOIN mto_milestone_solution_link AS link
+    ON m.id = link.milestone_id
+
+/* LEFT JOIN to see if the milestone's mto_common_milestone_key is
+     found in the common link table */
+LEFT JOIN mto_common_milestone_solution_link AS cLink
+    ON m.mto_common_milestone_key = cLink.mto_common_milestone_key
+
+WHERE
+    /* Only milestones whose model_plan_id is in the QIDS set */
+    m.model_plan_id IN (SELECT model_plan_id FROM QIDS)
```

Comment:
I think a join would be more appropriate here. We do it most places with a line like this (note you will need to adapt the names to conform since your CTE already has a name. 

`INNER JOIN QUERIED_IDS AS qIDs ON mto_milestone.model_plan_id = qIDs.model_plan_id`